pkg/policy/api/cidr.go

```diff
@@ -50,8 +50,9 @@ type CIDRRule struct {
 	ExceptCIDRs []CIDR `json:"except,omitempty"`
 
 	// Generated indicates whether the rule was generated based on other rules
-	// or provided by user
-	Generated bool `json:"-"`
+	// or provided by user. 0 indicates non-generated, >0 generated from a
+	// number of rules.
+	Generated uint `json:"-"`
```

Comment:
It's still odd to me that we have internal details like this exposed in the `api.CIDRRule` structure. I can't tell off-hand whether this will cause upgrade/downgrade problems but I guess the idea is that these are only used internally in-memory by the running Cilium instance, and hence shouldn't cause any problems.